index.js

```diff
@@ -116,35 +117,41 @@
     //
     // Returns tip or direction
     tip.direction = function(v) {
-      if (!arguments.length) return direction
-      direction = v == null ? v : d3.functor(v)
+      if (!arguments.length) {
+      	return direction;
+      }
+      direction = v == null ? v : functor(v);
 
-      return tip
-    }
+      return tip;
+    };
 
     // Public: Sets or gets the offset of the tip
     //
     // v - Array of [x, y] offset
     //
     // Returns offset or
     tip.offset = function(v) {
-      if (!arguments.length) return offset
-      offset = v == null ? v : d3.functor(v)
+      if (!arguments.length) {
+      	return offset;
+      }
+      offset = v == null ? v : functor(v);
 
-      return tip
-    }
+      return tip;
+    };
```

Comment:
Same, this looks like mostly formatting changes.  Could you collapse this like the previous version?